docs/usage/config-presets.md

```diff
@@ -16,6 +16,8 @@ In short:
 
 Shareable config presets can only be used with the JSON format, other formats are not supported.
 
+To learn more about the onboarding behavior read **insert proper link here once the `docs/development/shareable-configs.md` file has the final `h2` that we can link to**.
```

Comment:
TODO: insert proper link here once we have the final `h2` heading for the onboarding behavior explanation in `docs/development/shareable-configs.md`.